amivapi/tests/studydocs/test_summary.py

```diff
@@ -57,6 +64,11 @@ def test_filtered_summary(self):
 
         assert '_summary' in response
         assert response['_summary'] == {
+            'type': {
+                # The document with title `third` is ignored
```

Comment:
what does this coment say? 
is it actually the case that calling a study doc "third" will not work?

Reply:
The `where` clause used for the API request only matches documents where the title is `first` or `second`, so the document with the title `third` will not match the where condition.
Therefore the document is filtered out and should also not be counted in the summary part of the response.